ksql-engine/src/main/java/io/confluent/ksql/util/AvroUtil.java

```diff
@@ -86,16 +88,14 @@ private static boolean isValidAvroSchemaForTopic(
         return true;
       }
 
-      String errorMessage = e.getMessage();
       if (e.getStatus() == HttpStatus.SC_UNAUTHORIZED || e.getStatus() == HttpStatus.SC_FORBIDDEN) {
-        errorMessage = String.format(
-            "Not authorized to access Schema Registry subject: [%s]",
-            topicName + KsqlConstants.SCHEMA_REGISTRY_VALUE_SUFFIX
+        throw new KsqlSchemaAuthorizationException(
```

Comment:
Not sure how the ACLs work on SR, but we should correctly set the ACL in this exception so the user knows _what_ acls they need to set.